gradle.properties

```diff
@@ -11,4 +11,6 @@ POM_LICENCE_NAME=The Apache Software License, Version 2.0
 POM_LICENCE_URL=http://www.apache.org/licenses/LICENSE-2.0.txt
 POM_LICENCE_DIST=repo
 POM_DEVELOPER_ID=blipinsk
-POM_DEVELOPER_NAME=Bartosz Lipinski
\ No newline at end of file
+POM_DEVELOPER_NAME=Bartosz Lipinski
+android.useAndroidX=true
+android.enableJetifier=true
```

Comment:
why `android.enableJetifier=true`?

Reply:
Very strange, in my local changes this is false and git does not see any changes...